PyInstaller/building/imphookapi.py

```diff
@@ -85,11 +85,15 @@ def module_graph(self):
         "Current module graph"
         return self._module_graph
 
+    #FIXME: Remove this property and publicize the "_module_name" attribute to
+    #"module_name". This attribute is intended to be modifiable by hooks.
```

Comment:
Nowhere, Hartmut. Absolutely nowhere. Like all good things in life, you were just expected to know. (_Somehow._)

Nonetheless, it's the case. See this logic in the `PyiModuleGraph._safe_import_module()` method:

```
                # Respect method call changes requested by this hook.
                module_basename = hook_api.module_basename
                module_name = hook_api.module_name
```